comp/updater/ssistatus/fx/fx.go

```diff
@@ -0,0 +1,23 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2025-present Datadog, Inc.
+
+// Package fx provides the fx module for the ssistatus component
+package fx
+
+import (
+	ssistatus "github.com/DataDog/datadog-agent/comp/updater/ssistatus/def"
+	ssistatusimpl "github.com/DataDog/datadog-agent/comp/updater/ssistatus/impl"
+	"github.com/DataDog/datadog-agent/pkg/util/fxutil"
+)
+
+// Module defines the fx options for this component
+func Module() fxutil.Module {
+	return fxutil.Component(
+		fxutil.ProvideComponentConstructor(
+			ssistatusimpl.NewComponent,
+		),
+		fxutil.ProvideOptional[ssistatus.Component](),
```

Comment:
Since your comp has no public method nobody will depend on it and force its inclusion at init time.

You can force it's inclusion by adding a similar line: https://github.com/DataDog/datadog-agent/blob/main/comp/core/configsync/configsyncimpl/module.go#L50